bin/runtime-common/src/messages.rs

```diff
@@ -99,12 +99,52 @@ pub struct MessageTransaction<Weight> {
 	pub size: u32,
 }
 
+/// Helper trait for estimating the size and weight of a single message delivery confirmation
+/// transaction.
+pub trait TransactionEstimation<Weight> {
```

Comment:
One last thing - can you, please, change this name to something like `ConfirmationTransactionEstimation` or something like that? There's another (delivery) transaction here and it isn't "estimated" by this trait.

Reply:
Makes sense. Done.